code-challenges.test.js

```diff
@@ -14,39 +14,112 @@
+// divisibleByThree
+//   ✓ returns if the objects value is divisible by 3 (1 ms)
 
 // Explain your code:
+// I created a test that checks to see if a function called divisibleByThree takes an object and returns whether the value is divisible by 3. Then I created the function called divisibleByThree, set the parameter to take an object and returned the object.key is divisble by 3 using the modulus and setting it strictly equal to zero.
+
+
+
+
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
+
+
 // a) Create a test with expect statements for each of the variables provided.
+ describe("allCapitalized", () => {
+  const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+   // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+   const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+   // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+   it("returns an array with all the words capitalized", () => {
+     expect(allCapitalized(randomNouns1))
+     expect(allCapitalized(randomNouns2))
+   })
+ })
+// ReferenceError: allCapitalized is not defined
+
 
-const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
 // b) Create the function that makes the test pass.
+// Create a function that takes in an array of words
+// Outputs an array of words with all the words capitolized
+const allCapitalized = (arrayOfWords) => {
+  // Iterate value of the first character to uppercase
+  // Add the remaining values of the element to the Capitilized first value 
+  // Return the Array
+  return arrayOfWords.map(value => value[0].toUpperCase() + value.slice(1))
+}
+// PASS  ./code-challenges.test.js
+// allCapitalized
+//     ✓ returns an array with all the words capitalized
 
 // Explain your code:
+// I created the test to check to see if a function called allCapialized  that takes an array of words and returns those words with the first letter capitalized. I created the function called allCapitalized. Set the parameter to arrayOfNumbers and have it returning the parameter using the .map built in function. Within dot map I set the value of the first character per element to be uppercased with the toUpperCase method. It returned only an array of the first characters capitalized. I tried a few different ways to get the function to return the whole word. Using .filter and .map, using .slice trying to set the end point to arrayOfWords.length, adding the values together and eventually settled on .slice. Setting the starting value to 1 and not setting an ending value added the remainder of the characters of each item to the capitalized first letter.
```

Comment:
good work and nice explanation.  Your code is great, but your tests are incomplete. Check the template for Jest test and you'll see you're missing half of the expect statements.  This caused your #2 code to pass even though you did not get the expected output.  You're right there - just missed a step.